### ISSUE TITLE
Confirm hours before adding

### ISSUE THREAD

**Issue by mapomatic** [WazeUSA/WME-Place-Harmonizer#39]
Some places don't list am/pm and it's easy to copy/paste and forget about it.  We should add some way to verify that the hours being added are am/pm.

**Comment by RavenDT** [comment]
Good idea.  Maybe if there's any ambiguity, have a popup with what hours it plans on adding so you can just hit 'Yes' or 'No'.
